core/src/components/tabbar/tabbar.scss

```diff
@@ -77,10 +77,12 @@
 
 :host(.layout-icon-start) .tab-btn {
   --flex-direction: row;
```

Comment:
Since CSS values can not be assigned in a element that is not the host element `.tab-btn`, becuase we can refactor to:

```
:host(.layout-icon-start) {
  --flex-direction: row;
  --text-align: left;
}
```